dashboard/app/models/unit_group.rb

```diff
@@ -703,4 +703,9 @@ def self.latest_stable(family_name)
       sort_by(&:version_year).
       last
   end
+
+  def get_published_state
+    return course_version.published_state if course_version
+    published_state || SharedConstants::PUBLISHED_STATE.in_development
```

Comment:
I think I am missing some of the big picture here. could we `raise unless course_version`, and then just look at `course_version.published_state`? you don't have to raise here if this isn't the right place to do it, but I am wondering if we should be assuming at this point that all unit groups have course versions. if not, what's the scenario where a unit group does not have a course version?